src/window/new_career.c

```diff
@@ -66,7 +69,9 @@ static void handle_mouse(const mouse *m)
         window_go_back();
     }
```

Comment:
Shouldn't we `return` here?

Reply:
I added a `return` on the latest commit and it doesn't show up in the changes, has that happened to you before?